yarn-project/sequencer-client/package.json

```diff
@@ -68,5 +68,13 @@
   "types": "./dest/index.d.ts",
   "engines": {
     "node": ">=18"
+  },
+  "jest": {
+    "preset": "ts-jest/presets/default-esm",
+    "moduleNameMapper": {
+      "^(\\.{1,2}/.*)\\.[cm]?js$": "$1"
+    },
+    "testRegex": "./src/.*\\.test\\.(js|mjs|ts)$",
+    "rootDir": "./src"
```

Comment:
why is this needed?

Reply:
Master is soft-broken. It's broken if this component didn't cache

Reply:
this is running `yarn prepare`